### DISCUSSION TITLE
GitOpsCon EU 2022

### DISCUSSION THREAD

**Discussion by christianh814** [open-gitops/project#34] [General]
Work is starting already around GitOps Con EU 2022. A discussion was had on the fortnightly meeting and we are going to move forward to having it as a co-located event with KubeCon/CloudNativeCon again as we did with GitOps Con NA 2021 in Los Angeles.

This discussion is here as an async way of communicating and for transperency.

Anyone interested in joining the events committee can do it here on in the slack channel.

- [#wg-gitops-events](https://cloud-native.slack.com/archives/C01R2GXGTDL) Slack Channel

**Comment by roberthstrand** [comment]
As always, I want to help as much as time permits. However, I just don't have enough time to be part of the entire thing but what I can do is help with CFP review like I've done before. This is also one event that I hope to travel to, so I could also be on location and ~yell at~ help people if I'm there.

**Reply by christianh814** [reply]
Sounds good. I'll keep you updated about planning and shout if we need help. I'll keep you posted about the CFP committee though!

**Comment by ckavili** [comment]
I am keen to help as well! (and the timezone and location are close to me 😇✨)

**Comment by mikecali** [comment]
Very keen to help/assist as well @christianh814 :)

**Comment by christianh814** [comment]
I've reached out the the CNCF. The process has started! 🙌

I'll keep all of you posted as I get more info.

**Reply by christianh814** [reply]
The CNCF just got back to me and they are in the process of finalizing details for the co-located events for KubeCon EU 2022. Co-located events follow a slightly shorter planning timeline than KubeCon+CloudNativeCon, so the CFPs and planning won't begin in earnest, until the new year (2022). I will keep everyone posted as more info comes down!

**Comment by nishadmehendale** [comment]
Count me in well.

**Comment by weeyin83** [comment]
Would be happy to help on this side of things, especially the EU event. 👍

**Reply by roberthstrand** [reply]
Hurrah, Sarah! 🥳

**Comment by lloydchang** [comment]
@christianh814 @scottrigby @todaywasawesome

❔ Do you jointly own [abstract doc: GitOps Panel at Kubecon](https://docs.google.com/document/d/1c73uKKqLrzqUzrqQv_dDWxMtk_rO27_eL8bSW_K41YY/edit)?

Regarding the word "founding", such as:
> [Join us for a panel discussion with members from several of the founding organizations to learn about GitOps, OpenGitOps, and future work.](https://docs.google.com/document/d/1c73uKKqLrzqUzrqQv_dDWxMtk_rO27_eL8bSW_K41YY/edit)

My understanding — based on comments from @moshloop and @roberthstrand in other contexts — is that GitOps Working Group's _continuous usage_ of the word "founding" is an unfairness issue.

From https://github.com/open-gitops/website/issues/22#issuecomment-888305511

@moshloop commented:
> I think "Founding members" isn't really fair to latecomers or representative of the contribution.

From https://github.com/open-gitops/website/issues/22#issuecomment-888385285

@roberthstrand commented:
> I agree with Moshe that just having founding members is a bit unfair, but I don't want the members section to become a big mess.

From https://github.com/open-gitops/website/pull/16#issuecomment-888552249

@moshloop commented:
> The charter is explicit in that this WG is a:
> 1. Time-Bound (In which case Founding and Current are not relevant)
> 2. Vendor-Neutral, By differentiating between "membership" levels by definition you are not neutral.

This relates to other contexts https://github.com/open-gitops/website/pull/16 and https://github.com/open-gitops/website/issues/22

Thank you.

**Reply by todaywasawesome** [reply]
@lloydchang I created the doc because I'm planning to submit a GitOps maintainer's panel discussion talk submission for Kubecon EU similar to what we did at Kubecon US. I'm not sure exactly what your concern is as nothing in the doc mentions founding members. Current maintainers can be found in the [Maintainer's doc](https://github.com/gitops-working-group/gitops-working-group/blob/main/MAINTAINERS).

This thread we're comment in is about planning GitOpsCon and is unrelated to submitting a maintainer's panel talk to Kubecon.

**Reply by lloydchang** [reply]
@todaywasawesome Got it.

I just made 1-word change in https://docs.google.com/document/d/1c73uKKqLrzqUzrqQv_dDWxMtk_rO27_eL8bSW_K41YY/edit

from:
> founding organizations

to:
> maintainers organizations

**Reply by lloydchang** [reply]
Thanks @todaywasawesome

**Comment by sbernheim** [comment]
I'd be happy to help however I can!
